### ISSUE TITLE
Using API to stop recording after page reload

### ISSUE THREAD

**Issue by thebordella** [muaz-khan/Chrome-Extensions#84]
I am using RecordRTC_Extension() API to control recording using HTML elements on web page. E.g.

var recorder = new RecordRTC_Extension();
document.getElementById('btn-start-recording').onclick = function() { 
   recorder.startRecording({
   enableTabCaptureAPI: true,
   }); 
});
document.getElementById('btn-stop-recording').onclick = function() { 
   recorder.stopRecording(function(blob){ 
     //do something with blob
   });
});

Works good! 

But, suppose user action causes page to reload due to design of web app. After page reloads, the above code runs again, but recorder.stopRecording() does not stop the extension from recording. Instead, recording just continues.

I guess that new instance of var recorder = new RecordRTC_Extension() is somehow the problem? How to re-connect to extension to trigger stop?

**Comment by muaz-khan** [comment]
You can fire `stopRecording` inside `onbeforeunload` event. E.g.

```javascript
window.onbeforeunload = function() {
    recorder.stopRecording();
};
```

**Comment by thebordella** [comment]
Yes, that will work to stop recording before page reloads. But I want recording to continue while page reloads, and only stop when user clicks my stop button. But after page reload, my stop button does not work because "var recorder = new RecordRTC_Extension()" does not connect to same "instance" of extension. Maybe I am missing something?

**Comment by muaz-khan** [comment]
That feature isn't implemented at the moment. I'll let you know when its available.
